packages/app-store/jitsivideo/zod.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from "zod";
+
+export const appKeysSchema = z.object({
+  jitsiHost: z.string().optional(),
```

Comment:
Removed default from here as that doesn't work currently in the way this schema is used.